BusinessServices/PlayerManager.cs

```diff
@@ -18,19 +18,28 @@ public class PlayerManager: Manager
         public List<Player> PlayerBots;
         public EventManager _eventManager;
 
+        public List<UserCommand> CommandQueue;
+
         private string[] _playerColors = new string[]
         {
             "blue",
             "red",
             "black",
-            "purple"
+            "purple",
+            "cyan",
+            "fuchsia",
```

Comment:
I would've never guessed that's how you spell fuchsia

Reply:
I know right? It's a bizarre spelling, AND an insult to Sia